app/page.tsx

```diff
@@ -13,7 +13,7 @@ import { alphaConversionProp } from "./alphaConversionProp.js";
 import { alphaConversionPred } from "./alphaConversionPred.js";
 import astToSmt2Prop from "./astToSmt2Prop.js";
 import astToSmt2Pred from "./astToSmt2Pred.js";
-import generateSMTScriptProp from "./generateSMTScriptProp.js";
+import { generateSMTScriptProp } from "./generateSMTScriptProp";
```

Comment:
I avoid default exports. This allows you to lock down the name of things (it feels more wrong to write `import foo as bar`). You can see renaming biting 'round `grammarProp !== propositional_logic` (harder to quickly find the file).